packages/webapp/src/pages/Activity.tsx

```diff
@@ -512,10 +512,10 @@ export default function Activity() {
                                 <ChevronsLeft onClick={resetOffset} className="flex stroke-white cursor-pointer mr-3" size="16" />
                             )}
                             <span onClick={decrementPage} className={`flex ${offset - limit >= 0 ? 'cursor-pointer hover:bg-gray-700' : ''} h-8 mr-2 rounded-md px-3 pt-1.5 text-sm text-white bg-gray-800`}>
-                              <svg aria-hidden="true" className="w-5 h-5" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" d="M7.707 14.707a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414l4-4a1 1 0 011.414 1.414L5.414 9H17a1 1 0 110 2H5.414l2.293 2.293a1 1 0 010 1.414z" clip-rule="evenodd"></path></svg>
```

Comment:
I don't know much about svg. Does it change anything or is it simply to be consistent

Reply:
You will receive a warning: Invalid DOM property `fill-rule.` Did you mean `fillRule`? implying that one needs to use `camelCase` for SVG attributes.